xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/servletEngine/ServletContainerExecutor.java

```diff
@@ -125,8 +129,7 @@ public void start(File sourceWARDirectory) throws Exception
                 this.jettyStandaloneExecutor.start();
                 break;
             default:
-                throw new RuntimeException(String.format("Servlet engine [%s] is not yet supported!",
-                    this.testConfiguration.getServletEngine()));
+                this.throwExceptionForServletEngineNotYetSupported(testConfiguration);
```

Comment:
I don't understand the need for this method. Doesn't seem it's reused and it's only 1 line. This makes it harder to read unnecessarily to me.

Reply:
I see. IMO the issue is that you split the switch into 2. I feel it would be more readable/more logical to have only 1 switch.

Reply:
Agree